pkg/asset/agent/image/ignition.go

```diff
@@ -499,7 +499,16 @@ func RetrieveRendezvousIP(agentConfig *agent.Config, nmStateConfigs []*v1beta1.N
 	} else {
 		err = errors.New("missing rendezvousIP in agent-config or at least one NMStateConfig manifest")
```

Comment:
nit: I'd be inclined to `return "", err` here rather than check for the error later.

Reply:
> nit: I'd be inclined to `return "", err` here rather than check for the error later.

Changed to return earlier.